adapters/between/betweentest/exemplary/multi-request.json

```diff
@@ -0,0 +1,160 @@
+{
+  "mockBidRequest":
+  {
+    "id": "some-request-id",
+    "imp": [
+      {
+        "id": "some-impression-id",
+        "banner":
+        {
```

Comment:
I'd suggest adding another imp here with different `host` value as that is how you're clubbing the imps together for each request to your adapter

Reply:
I'm in doubt we'll have different hosts in an impressions near future